src/main/java/emu/grasscutter/game/quest/QuestManager.java

```diff
@@ -386,6 +456,10 @@ public void checkQuestAlreadyFullfilled(GameQuest quest){
                             queueEvent(condition.getType(), condition.getParam()[0], condition.getParam()[1]);
                         }
                     }
+                    case QUEST_CONTENT_GAME_TIME_TICK -> {
+                        // might need extra condition checking
+                        queueEvent(condition.getType(), condition.getParam()[0], condition.getParam()[1]);
```

Comment:
Wouldn't this always succeed, I depend of the time?

Reply:
Im only adding this here as a work around for 358, but i guess unlike the other ones, the condition checking in the exec file would be sufficient for the checking